libraries/AP_Notify/AnyLED_I2C.cpp

```diff
@@ -0,0 +1,87 @@
+/*
+   AnyLED I2C driver
+*/
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "AnyLED_I2C.h"
+#include <AP_HAL/AP_HAL.h>
+#include <utility>
+
+
+extern const AP_HAL::HAL& hal;
+
+#define ANY_LED_BRIGHT      0xFF    // full brightness

```

Comment:
These are misleading, it's identical brightness.... Does the driver not support variable brightness?

Reply:
yes, the identical brightness, let the external led controller to control the brightness, different people   may have different LEDs , I think the RGB values for external led controllers is enough, I just want to make the external led more interesting and may not lost the main LED status for Copter.

Reply:
so, this driver is to control external LEDs except you want the external system to actually control them?

Reply:
yes, my idea is the external system to actually control external LEDs,  For the external system, the important  is the RGB values from FC. For example, we defined the different colors and styles for UAV status, but for many people, many colors and styles may complex, they only want two status of UAV,  system normal or system exception,  so from this "AnyLED_I2C" they can make external LEDs custom color and style.